lib/api/BackbeatAPI.js

```diff
@@ -86,7 +86,7 @@ class BackbeatAPI {
      */
     validateQuery(bbRequest) {
         const { marker } = bbRequest.getRouteDetails();
-        if (marker !== undefined && (marker === '' || isNaN(marker))) {
+        if (marker !== undefined && Number.isNaN(Number.parseInt(marker, 10))) {
```

Comment:
Playing around with `isNaN` and `Number.parseInt`, looks like validation was stronger in the previous implementation, e.g. `parseInt('3abc')` returns 3, while `isNaN('3abc')` returns true.

Reply:
Nice catch, I did not now about this case. I'll stick with the global `isNaN`.